entrypoint.sh

```diff
@@ -1,5 +1,19 @@
 #!/bin/bash
 
+#Select the out put format
+if [ "${TFSEC_OUTPUT_FORMAT}" != "" ]; then
```

Comment:
If you put `default: default` here, you will never have this if `false` or do I miss understand something?

Reply:
good catch, ill redo the logic somehow

Reply:
Because it always has a default value, i took out the check